DemoEdgeApp/ModuleWrapper/MockModuleClientProxy.cs

```diff
@@ -0,0 +1,102 @@
+﻿
+namespace ModuleWrapper
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Amqp.Framing;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Shared;
+    using Serilog;
+
+    public class MockModuleClientProxy : IModuleClient
+    {
+        public Dictionary<string, List<Message>> MessageQueues { get; private set; }
```

Comment:
You can omit the private set from properties if you only want a public setter (it will be generated for you)

Reply:
But if she wants to prevent it from being set externally then she is good with a private setter

Reply:
as above